domains/eternalrainfall.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "SmartArt09",
+           "email": "",
+           "discord": "698751040010321990",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.B4uAzpuIM3qAOxKrX2z04Dqu21PsVVZmiKqxWaV7t922tOakxpR2sHgN5DCAV1mwIKD1ZRdUpnU9OhuwgGBr_Apf3hlrmntnaZ9oZ7tILHQ4J3AOeGP-ctBca--wU89OBY4iQ2ocjHHJygDwlZ1PNP1_iSfhaZA6ii_sBWMwliYvd6ULcn-xuc1YSdW8i1aF5Y3JNWLtwVK8Xd_GXsRjmobz7HS3sE644PA_qr9F4gYCnYtoEfQEc4yO90Hor8_I_CRm1YkTWA1qKgpj5_MqjDNv1kJzVhT8kvb6hiHSTdHvHuN8GU5qs_1K19j5ZgXpmMd-pD0FPMdCSq8S5kCHvw.ylfEN1SY-DonOTr2aIHnXg.LREkjvwcgkTEhAAJXAxEgQuI7h-8T7MYZXtlrQdRKwL_uy9Do36FAvN2hakEGtR8WvEjEA2MCGteScFD_eVSCUIIzsTm3_M7WniweMBrijo.eA3X665dGOht3ECUm4EXHQ"
+        },
+    
+        "record": {
+            "TXT": "8a4df5518629977b17ebe10d7238db"
```

Comment:
TXT records alone doesn't do anything.

Reply:
Sorry, first time doing this, how and what do I need to change or add?

Reply:
You would need to have records like CNAME or A, something that displays a site. TXT records are only used for verification, and we haven't seen any reason for this domain to be verified.